src/pretalx/mail/context.py

```diff
@@ -46,9 +46,8 @@ def template_context_from_submission(submission):
         ),
         'event_name': submission.event.name,
         'submission_title': submission.title,
-        'submission_url': build_absolute_uri(
-            'cfp:event.user.submission.edit',
-            kwargs={'event': submission.event.slug, 'code': submission.code}
-        ),
+        'submission_url': submission.urls.user_base.full(scheme='https'),
```

Comment:
optional side-note: `PRETALX_SITE_URL` and similar settings allow a full base URL with protocol to be specified. While I agree that everything should be https://, because 2017, this does not honor the settings.

Reply:
I don't quite agree – we have strong warnings against http sites in our documentation, and development urls are invalid in any case atm because we don't try to add the correct port number.

I'll disregard this for now because I'll add a custom Url class in a few days which will have all the `full()` logic in one place and can be clever enough to figure this out correctly.